resources/views/snippets/link_table.blade.php

```diff
@@ -26,7 +26,22 @@
                 {{str_limit($link->long_url, 50, '...')}}
             </a>
         </td>
-        <td>{{$link->clicks}}</td>
+        <td>
+            <div class="btn-group">
+                <button type="button" class="btn btn-sm btn-info"><span class="badge">{{count($link->clicks)}}</span></button>
+                <button type="button" class="btn btn-sm btn-info dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                    <span class="caret"></span>
+                    <span class="sr-only">Toggle Dropdown</span>
+                </button>
+                <ul class="dropdown-menu">
+                    <li><a href="" ng-click="showStats($event, '{{route('stats', ['link_id' => $link->id, 'type' => 'day'])}}')">Clicks by day</a></li>
+                    <li><a href="" ng-click="showStats($event, '{{route('stats', ['link_id' => $link->id, 'type' => 'country'])}}')">Clicks by country</a></li>
+                    <li><a href="" ng-click="showStats($event, '{{route('stats', ['link_id' => $link->id, 'type' => 'referer'])}}')">Clicks by referer</a></li>
+                    <li role="separator" class="divider"></li>
+                    <li><a href="{{route('clicks', ['link_id' => $link->id])}}">List all</a></li>
```

Comment:
This could be moved to the header or footer of a modal with all the graphs combined.